CIP-0381/README.md

```diff
@@ -1,24 +1,25 @@
 ---
 CIP: 0381
 Title: Plutus support for Pairings over BLS12-381
-Authors: Iñigo Querejeta-Azurmendi <inigo.querejeta@iohk.io>
-Discussions-To: https://github.com/cardano-foundation/CIPs/pull/220
-Comments-URI: https://github.com/cardano-foundation/CIPs/pull/220
-Category: Plutus
 Status: Proposed
-Type: Standards Track
+Category: Plutus
+Authors:
+  - Iñigo Querejeta-Azurmendi <inigo.querejeta@iohk.io>
+Implementors: N/A
```

Comment:
```suggestion
Implementors:
  - IOG
```

Reply:
IOG or IOG Plutus team?

Reply:
@Ryun1 I would be happy with @michaelpj's preference about which to put there.  But can we call them Implementors without being sure that they've fully implemented it?

Reply:
It's implemented.

In the past we've mostly put individuals here. I suggest: `kenneth.mackenzie@iohk.io` (@kwxm), since he did the work.

Reply:
... done (https://github.com/cardano-foundation/CIPs/pull/719/commits/a7840d93e7d57df73faaf031cd0b96118496d0fd)